test/rules/structured-imports/ancestorCases-fix.ts.lint

```diff
@@ -0,0 +1,6 @@
+import * as SomethingAbsolute from 'something-absolute';
```

Comment:
Ahhh, my bad. It looks for a file called `someTestName.ts.fix`. I named the files wrong.